find.go

```diff
@@ -55,7 +55,7 @@ func Find(in, out, predicateFn interface{}) error {
 				return nil
 			}
 		}
-
+		return fmt.Errorf("element not found")
```

Comment:
@aswinkarthik Should we return error if the element is not found?

Reply:
Yes. In order to differentiate can we have a separate ErrElementNotFound so that it can be distinguished from validation errors?

Reply:
Okay. I agree. @flyznex Could we do it as part of this PR? What do you think?

Reply:
@dineshba I think we need create new file for define, format errors. In this PR, ours issue has been solved. We will create separate Issue, PR for error handling.

Reply:
Sure @flyznex